lib/express-rate-limit.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 var defaults = require('defaults');
+var MemoryStore = require('./memory-store');
 
 function RateLimit(options) {
 
     options = defaults(options, {
         // window, delay, and max apply per-key unless global is set to true
-        windowMs: 60 * 1000, // milliseconds - how long to keep records of requests in memory
+        windowMs: 60 * 1000, // milliseconds - how long to keep records of requests in memory: Only used when store is MemoryStore
```

Comment:
I think we should omit this addition... presumably other stores could also use windowMs if they wanted.